molo/core/templatetags/core_tags.py

```diff
@@ -24,34 +24,25 @@ def get_language(site, locale):
     return language
 
 
-def get_pages(context, qs, locale):
+def get_pages(context, queryset, locale):
     request = context['request']
+    pages = []
+    if not hasattr(request, 'site'):
+        return pages
```

Comment:
This will always return an empty list -- did you want to return the original queryset?